tezos/context/src/working_tree/storage.rs

```diff
@@ -1400,36 +1414,36 @@ impl Storage {
 
     pub fn clear(&mut self) {
         if self.blobs.capacity() > 2048 {
-            self.blobs = Vec::with_capacity(2048);
+            self.blobs = ChunkedVec::with_chunk_capacity(2048);
```

Comment:
These numbers repeat, and are also different from the current defaults set in `Storage::new` which I guess was not intended, so I think it would be good to define constants for them.